basie/tests/test_schedule.py

```diff
@@ -10,13 +10,16 @@
 from basie import target_parser
 
 BASE_PATH = ".basie_test"
+curdir = os.path.abspath(os.path.dirname(__file__))
```

Comment:
I changed all paths to data files to be relative to the current directory but without needing soft links (which would not work on Windows, by the way)